backend/src/nodes/utils/clipboard/clipboard_win32.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import win32clipboard 
+
+from .clipboard_base import ClipboardBase
+
+class WindowsClipboard(ClipboardBase):
+    def __init__(self) -> None:
+        if win32clipboard is None:
+            raise Exception("pywin32 must be installed to use this library on Windows platform.")
+
+    def copy_image(self, img: bytes) -> None:
+        win32clipboard.OpenClipboard()
+        win32clipboard.EmptyClipboard()
+        win32clipboard.SetClipboardData(49472, img)  # type: ignore # 49472 -> PNG format
```

Comment:
Could you make this a constant at the top of the file rather than a magic number?

Reply:
Thanks for the feedback.
Updated all the mentioned issues, hopefully it all fits now.